app/buckets/handlers.js

```diff
@@ -42,7 +42,8 @@ exports.bucket_details = (req, res, next) => {
         bucket,
         apps_options: apps.exclude(bucket.apps),
         users_options: users.exclude(bucket.users),
-        current_user_is_bucket_admin
+        signedInUserId: req.user.auth0_id,
```

Comment:
I wonder if we should call this variable `current_user_id` for consistency with similar variables (with the snake case like the other surrounding variables)

Reply:
You don't need to pass this to the template at all - it is already available as `current_user.auth0_id`

Reply:
👍